Automating newsletter issue (#115)
pythonpune/meetup-talks: .github/ISSUE_TEMPLATE/newsletter_box.md

---

> name: Newsletter

Reviewer: Let's call it 'Newsletter collection box'?

Author: sure

> title: "{{ env.EVENT }} PythonPune Newsletter Collection Box"

Reviewer: ```suggestion
title: "{{ env.EVENT }} PythonPune newsletter collection box"
```

labels: newsletter
---

> Let's use this issue to collect python related news, new package updates and events happening around Pune.

Reviewer: ```suggestion
Let's use this issue to collect Python related news, new package updates and events happening around Pune.
```

We can compile and publish it during **{{ env.EVENT }} PythonPune** meetup. 

### How to add news:

> Comment on this issue with news link and news topic and also include the source from we got this. 

Reviewer: ```suggestion
Comment on this issue with news link. Please try to include the source like tweet / reddit and a TL;DR section giving brief about the link.
```
